### PR TITLE
log failed to get access token

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: TA-microsoft-graph-security-score/bin/input_module_graph_security_score_input.py [hunk 1]
```diff
@@ -36,13 +36,16 @@ def get_access_token(helper, application_id, secret, tenant):
...
+        )
     else:
-        access_token = helper.send_http_request(
+        resp = helper.send_http_request(
             url, "POST", payload=urllib.urlencode(data), timeout=(15.0, 15.0)
-        ).json()
+        )
+    if resp.status_code not in (201, 200):
+        helper.log_error("Failed to get access_token. status_code={}, resp={}".format(resp.status_code, resp.text))
+    access_token = resp.json()
```
Comment (VatsalJagani) [inline TA-microsoft-graph-security-score/bin/input_module_graph_security_score_input.py:49]: This would generate an error in case of error code not ok. ??

Comment (mahirchavda) [reply TA-microsoft-graph-security-score/bin/input_module_graph_security_score_input.py:49]: yes

Comment (VatsalJagani) [reply TA-microsoft-graph-security-score/bin/input_module_graph_security_score_input.py:49]: Should we manually raise Exception("Unable to get the access_token + response_code and response.text")??

Comment (mahirchavda) [reply TA-microsoft-graph-security-score/bin/input_module_graph_security_score_input.py:49]: Updated using resp.raise_for_status()

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**